publication-commons/src/main/java/no/unit/nva/publication/model/business/FileEntry.java

```diff
@@ -252,9 +254,10 @@ private boolean pendingFileTypeIsUpdated(File file) {
                && !this.file.getArtifactType().equals(file.getArtifactType());
     }
 
-    public void importNew(ResourceService resourceService,UserInstance userInstance, ImportSource importSource) {
+    public void importNew(ResourceService resourceService, UserInstance userInstance, ImportSource importSource) {
         this.modifiedDate = Instant.now();
-        this.setFileEvent(FileImportedEvent.create(userInstance.getUser(), Instant.now(), importSource));
+        this.setFileEvent(FileImportedEvent.create(userInstance.getUser(), userInstance.getTopLevelOrgCristinId(),
+                                                   Instant.now(), importSource));
```

Comment:
User when importing file is dummy value, failing on fetching this user from identitiy-service when creating log message.